flac.md

```diff
@@ -35,7 +24,7 @@ The FLAC format, the FLAC reference implementation and this document were origin
 
 - **Frame**: A frame header, one or more subframes and a frame footer. It encodes the contents of a corresponding block.
 
-- **Subframe**: An encoded subblock. All subframes within a frame code for the same number of samples. A subframe MAY correspond to a subblock, else it corresponds to either the addition or subtraction of two subblocks, see [section on interchannel decorrelation](#interchannel-decorrelation).
+- **Subframe**: An encoded subblock. All subframes within a frame code for the same number of samples. A subframe correspond to a subblock, the average of two subblocks or the difference between two subblocks, see [section on interchannel decorrelation](#interchannel-decorrelation).
```

Comment:
Should it say 'A subframe corresponds to a subblock'?

Reply:
Yes, thanks

Reply:
```suggestion
- **Subframe**: An encoded subblock. All subframes within a frame code for the same number of samples. A subframe corresponds to a subblock, the average of two subblocks or the difference between two subblocks, see [section on interchannel decorrelation](#interchannel-decorrelation).
```